.github/workflows/ci.yml

```
@@ -0,0 +1,459 @@
+name: CI
+on:
+  workflow_call:
+    inputs:
+      # extra jobs must be multi-line string, as there's no support for `type: array` for inputs
+      extra_jobs:
+        type: string
+        required: false
+        default: ''
+      composer_require_extra:
+        type: string
+        required: false
+        default: ''
+      dynamic_matrix:
+        type: boolean
+        default: true
+      simple_matrix:
+        type: boolean
+        default: false
+      endtoend:
+        type: boolean
+        default: true
+      phpcoverage:
+        type: boolean
+        default: false
+      phplinting:
+        type: boolean
+        default: true
+      phpunit:
+        type: boolean
+        default: true
+      js:
+        type: boolean
+        default: true
+
+jobs:
+
+  context:
+    name: Context
+    runs-on: ubuntu-latest
+    env:
+      # Put this through an env variable to prevent possible string substitution injection via pull-request branch name
+      GITHUB_HEAD_REF: ${{ github.head_ref }}
+    steps:
+      - name: Context
+        run: |
+          # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
+          # github.base_ref - The target branch of the pull request in a workflow run.
+          echo "github.base_ref: ${{ github.base_ref }}"
```

```suggestion
          echo "Target branch: ${{ github.base_ref }}"
```
Lets just include the comment in the echo - that way when debugging we won't have to come back to this src to check what the workflow is telling us about the context.

---

I don't like this, it just adds an extra step to debugging. The point of this is specifically to help work out what github context variables point to what.

---

I misunderstood the purpose of the output then - just add a comment saying something like "output the github context variables so that we know what each of them is pointing to"